docs/lab/README.md

```diff
@@ -0,0 +1,93 @@
+# Kernel Module Management Lab 
+
+Kernel Module Management a.k.a KMM manages out of tree kernel modules in Kubernetes.
+
+By installing the controller in a Kubernetes cluster we will get a new Namespace where a Deployment will run and create a ReplicaSet.
+
+Once the ReplicaSet is running we can create and use a new Module object which will manage the build and/or run of a kernel module.
+
+In this document we will find examples and YAML files including in-cluster module building, device plugins and use of module loaders with [KMM](https://github.com/kubernetes-sigs/kernel-module-management) for different Kubernetes environments : [EKS](aws), [GKE](gce), [Azure](azure), [Ubuntu bare metal](ubuntu). 
+
+## Installation
+
+### Command line:
+
+``kubectl apply -k https://github.com/kubernetes-sigs/kernel-module-management/config/default``
+
+### Operator Hub
+
+TO-DO
+
+## Platforms :
+- EKS Cluster in Amazon Web Services
+- GKE Cluster in Google Cloud Platform
+- Vanilla Kubernetes with Ubuntu Nodes
+- AKS Cluster in Microsoft Azure
+
+###  [Elastic Kubernetes Service (EKS)](aws)
```

Comment:
Link is broken and probably redundant as you linked them in the top part of this document.

Reply:
I am pretty sure you should be able to just get the link of a headline in markdown.
![image](https://user-images.githubusercontent.com/29724509/188818205-ed9bca9e-1f9b-4933-a9ca-b64d77afe1a7.png)